src/channelape/controller/ChannelApeActionsController.ts

```diff
@@ -10,36 +10,36 @@ const PARSE_INT_RADIX = 10;
 
 export default abstract class ChannelApeActionsController {
```

Comment:
Please change this controller so that the only protected function is `processAction` and the only public method is `handle`. We should hide `complete` and `handleError` within `handle` so that web services do not need to worry about making those calls in their controllers.